cdc/sinkv2/eventsink/txn/mysql/mysql.go

```diff
@@ -93,6 +93,12 @@ func NewMySQLBackends(
 	if err != nil {
 		return nil, err
 	}
+
+	cfg.IsTiDB, err = isTiDB(ctx, db)
```

Comment:
If the BDR is enabled , can we pre-check whether the downstream is tidb

Reply:
done